projects/angular/src/utils/_mixins.scss

```diff
@@ -230,3 +231,136 @@ $opp-directions: (
     @content;
   }
 }
+
+@function baselinePx($pixels: 0) {
+  @return calc($pixels * (1rem / #{tokens.$cds-global-base}));
+}
+
+@mixin generate-typography-header-token($name, $addFontFamily: false, $addColor: false) {
+  @if ($addColor) {
+    color: #{tokens.$cds-alias-typography-color-300};
+  }
+
+  @include generate-typography-token($name, $addFontFamily);
+}
+
+@mixin generate-typography-body-token($name, $addFontFamily: false, $addColor: false) {
+  @if ($addColor) {
+    color: #{tokens.$cds-alias-typography-color-450};
+  }
+
+  @include generate-typography-token($name, $addFontFamily);
+}
+
+@mixin generate-typography-token($name, $addFontFamily: false) {
+  @if ($addFontFamily) {
+    font-family: #{tokens.$cds-global-typography-font-family};
+  }
+
+  @if ($name == 'DISPLAY-40') {
+    // H1, Login Header
+    font-size: #{tokens.$cds-global-typography-font-size-9};
```

Comment:
These values should reference the alias variables (`$cds-alias-typography...`).

These should be mapped to their respective variables (such as `$cds-alias-typography-display-font-size`) rather than the underlying value. 

A `map` may also be a better, more organized way to store/retrieve this information.

Reply:
I didn't know those tokens existed. I'll  try to convert to `map` too.

Reply:
Using `map` for data structure. Used `alias` based tokens at all possible locations. Some don't have concrete tokens.
Changes are made in commit 585c5f2.